hardhat.config.ts

```diff
@@ -35,6 +39,103 @@ task(
   }
 )
 
+task("collateral", "Mint Collateral Tokens for use in tests")
```

Comment:
I feel like this file is getting a little large and unruly, have you seen any documentation on breaking this out?

Reply:
I agree. After 20min of research I did not find an obvious / recommended way how to split this file so for now I just moved the tasks to the end of the file to hopefully improve readability a little bit.